### PR TITLE
Make Navbar Responsive

### LINKED ISSUE #177: Add Mobile Navbar

### LINKED ISSUE #190: Navbar changes
- [x] The font in the header is too big, should be smaller like in last year's website. These are prints from the same screen and everything is much larger.

Captura de ecrã 2022-01-10, às 11 16 40

Captura de ecrã 2022-01-10, às 11 18 16

- [x] Moonstone should be called login

- [x] Team page will not be available for now so it should temporarily be removed from the header

### PR DESCRIPTION
**Pull request by feliciofilipe**
Closes #177 
Closes #190

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -3,18 +3,19 @@ import Sponsors from '/components/website/home/Sponsors';
...
   return (
-    <>
-      <Hero />
-      <Schedule color="tertiary" detailed={false}/>
-      <Sponsors/>
-      <Hackathon />
-      <Speakers />
-      <Footer color="secondary" />
-    </>
+      <Navbar bg_color="secondary" button="quinary" fg_color="black">
```
Comment (nelsonmestevao) [inline pages/index.js:12]: In JavaScript, variables (in this case props) should be camelCase.
```suggestion
      <Navbar bgColor="secondary" button="quinary" fgColor="black">
```

**Review by miguelbrandao — DISMISSED** [review]

**Review by ruioliveira02 — CHANGES_REQUESTED** [review]
image
Please fix this warning and solve the conflict.

**Review by mariajbp — APPROVED** [review]

File: components/website/utils/Navbar/index.jsx [hunk 2]
```diff
@@ -1,40 +1,94 @@
...
+
+function classNames(...classes) {
+    return classes.filter(Boolean).join(' ')
+}
+
 const navigation = [
     { name: 'Schedule', href: '/schedule' },
     { name: 'Hackathon', href: '/hackathon' },
-    { name: 'Team', href: '/team' },
+    // { name: 'Team', href: '/team' },
```
Comment (nelsonmestevao) [inline components/website/utils/Navbar/index.jsx:15]: Keep in mind that this doesn't prevent the user to access `/team`.

**Comment by nelsonmestevao** [comment]
image

There's a styling bug when you open navbar in mobile and go back to desktop version. Not a big problem though but shouldn't be hard to fix with a `lg:`.

**Comment by nelsonmestevao** [comment]
> Shouldn't there be an option to sign up to moonstone via the mobile navbar?

Anyway, I think this button is not to sign up in moonstone, but to register on Event Brite. When you do that, you receive an account. @mariajbp @miguelbrandao @jtexeira should check this flow.

Last year's edition the button in mobile went down:

image

File: components/website/utils/Navbar/index.jsx [hunk 3]
```diff
@@ -1,40 +1,94 @@
...
     { name: 'Challenges', href: '/challenges' },
     { name: 'Speakers', href: '/speakers' },
     { name: 'FAQs', href: '/faq' },
-    { name: 'Moonstone', href: '/login' },
+    { name: 'Login', href: '/login' },
 ]
 
-export default function Navbar({ button, fg_color }) {
+export default function Navbar(props) {
```
Comment (nelsonmestevao) [inline components/website/utils/Navbar/index.jsx:22]: Checkout https://2021.seium.org/ to see the mobile animation that we are missing.

**Comment by miguelbrandao** [comment]
You are right @nelsonmestevao ,there is no sign-up, only the registration on moonstone

File: components/website/utils/Navbar/index.jsx (hunk 2 above)
Comment (feliciofilipe) [reply components/website/utils/Navbar/index.jsx:15]: Yeah, what should I do in this regard?

Comment (feliciofilipe) [reply components/website/utils/Navbar/index.jsx:15]: I see, thanks :pray::pray::pray:

**Comment by feliciofilipe** [comment]
@miguelbrandao @nelsonmestevao I am adding to the Join us button on the mobile version as in the previous edition

**Comment by feliciofilipe** [comment]
> image
> 
> There's a styling bug when you open navbar in mobile and go back to desktop version. Not a big problem though but shouldn't be hard to fix with a `lg:`.

I am already doing this, I think its another situation on params in tailwind classes not working always

**Comment by feliciofilipe** [comment]
@nelsonmestevao 

I think now its good and it looks really nice

<details>
<summary><strong>Previews</strong></summary>

image
image
image

</details>

**Review by miguelbrandao — APPROVED** [review]

**Review by rubenadao — APPROVED** [review]

**Review by ruioliveira02 — APPROVED** [review]

**Pull request merged**